deployment/docker-compose.integration.yml

```diff
@@ -0,0 +1,38 @@
+version: "3.8"
+
+services:
+  redis:
+    image: redis:6
+    # Set health checks to wait until redis has started
+    healthcheck:
+      test: ["CMD", "redis-cli", "ping"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+
+    ports:
+      - 6379:6379
+
+  postgres:
+    image: postgres:13
+    # Set health checks to wait until postgres has started
+    healthcheck:
+      test: ["CMD", "pg_isready"]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+    environment:
+      # postgres image requires password to be set
+      PGUSER: ${PGUSER}
```

Comment:
where is this var set?

Reply:
good catch, forgot to add it to `.env.example`